updating/update-using-custom-machine-config-pools.adoc

```diff
@@ -0,0 +1,76 @@
+* performing the cluster update
+* removing the custom labels to move some nodes back to their original MCP which triggers the update on those nodes
+* testing the applications on those nodes to make sure the applications work as expected on those newly-updated nodes
+* removing the custom labels from the remaining nodes in small batches and testing the applications on those nodes 
+
+//The following wording comes from https://github.com/openshift/openshift-docs/pull/34704, not yet finalized
+
+[NOTE]
+====
+Pausing a machine config pool prevents the Machine Config Operator from applying any configuration changes on the associated nodes. Pausing an MCP also prevents any automatically-rotated certificates from being pushed to the associated nodes, including the automatic CA rotation of the `kube-apiserver-to-kubelet-signer` CA certificate. If the MCP is paused when the `kube-apiserver-to-kubelet-signer` CA certificate expires and the MCO attempts to automatially renew the certificate, the new certificate is created but not applied across the nodes in the respective machine config pool. This causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. Pausing an MCP should be done with careful consideration about the `kube-apiserver-to-kubelet-signer` CA certificate expiration and for short periods of time only.  
+====
+
+To ensure that the cluster remains stable, creating a custom MCP for the control plane nodes (also known as the master nodes) is not supported. The Machine Config Operator (MCO) ignores any custom MCP created for the control plane nodes.
+
+[id="update-using-custom-machine-config-pools-about-mcp_{context}"]
+== About the update process and MCPs
+
+In {product-title} nodes are not considered individually. Nodes are grouped into machine config pools(MCP). There two MCPs in a default {product-title} cluster: one for the control plane nodes and one for the worker nodes. An {product-title} update affects all MCPs concurrently.
+
+During the update, the Machine COnfig Operator (MCO) drains and cordons all nodes within a MCP up to the specified `maxUnavailable` number of nodes (if specified). Draining and cordoning a node deschedules all pods on the node and marks the node as unschedulable. Once drained the Machine Config Daemon applies a new machine configuration during the update, which can include updating the operating system, which requires the host to reboot. 
+
+To prevent specific nodes from being updated, and thus, not drained, cordoned, and updated, you can create custom MCPs to ensure that the nodes are not updated. The MCO does not update any paused MCPs. You can create one or more custom MCPs, which can give more control over the sequence in which nodes are updated. After the update of the first MCP, the application compatibility can be verfied and then the whole fleet can be updated gradually to the new version.
```

Comment:
> To prevent specific nodes from being updated, and thus, not drained, cordoned, and updated, you can create custom MCPs to ensure that the nodes are not updated

it should be ` you can pause the custom MCPs to ensure that the nodes are not updated`